app/src/main/java/com/github/se/polyfit/ui/components/card/IngredientInfoCard.kt

```diff
@@ -0,0 +1,66 @@
+package com.github.se.polyfit.ui.components.card
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.IntrinsicSize
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Card
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.github.se.polyfit.R
+import com.github.se.polyfit.model.ingredient.Ingredient
+
+@Composable
+fun IngredientInfoCard(ingredient: Ingredient) {
+  Card(
+      onClick = {},
```

Comment:
### Question
- The onClick parameter for the Card is currently empty. Is there a future plan to add functionality here, or should it be removed if not needed?

Reply:
The `onClick` is empty because nothing should happen if the user clicks on the card. But yeah you're right I will delete it to make the code clearer.